### PR TITLE
Dashboard Prototype Framework 

### PR DESCRIPTION
**Pull request by Obinnabii**
- Create a dashboard page
- Add Bootstrap 4 tab functionality. I plan to use tabs to simplify how the mentors view mentees and projects.
- Add some minor styling to the tab headers.

### CODE REVIEW

File: src/main/webapp/dashboard.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" data-toggle="tab" href="#int-mentees">Interested Mentees</a>
+    </li>
+  </ul>
+
+  <!-- Tab panes -->
+  <div class="tab-content">
+    <!-- About Me -->
+    <div id="abt-me" class="container tab-pane active"><br>
```
Comment (Richie78321) [inline src/main/webapp/dashboard.html:32]: Should probably avoid abbreviations where possible to preserve readability. Could change to `about-me`.

File: src/main/webapp/dashboard.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <h3>About Me</h3>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+    </div>
+    <!-- Affiliated Projects -->
+    <div id="affiliations" class="container tab-pane fade"><br>
```
Comment (Richie78321) [inline src/main/webapp/dashboard.html:37]: This ID could be more specific. How about something like `project-affiliations`?

File: src/main/webapp/dashboard.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      <h3>Affiliations</h3>
+      <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+    </div>
+    <!-- Interested Mentees -->
+    <div id="int-mentees" class="container tab-pane fade"><br>
```
Comment (Richie78321) [inline src/main/webapp/dashboard.html:42]: Same comment as above. Should avoid the abbreviation with `interested-mentees`.

File: src/main/webapp/style.css [hunk 4]
```diff
@@ -1,8 +1,14 @@
 h1 {
-  font-family: Verdana, sans-serif; 
+  font-family: Verdana, sans-serif;
   font-size: x-large;
 }
```
Comment (Richie78321) [inline src/main/webapp/style.css:4]: Consider creating a class for this styling to [avoid using type selectors](https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors) or consider applying the font to a more global element like `body` to keep the font consistent between different text elements.

File: src/main/webapp/style.css [hunk 5]
```diff
@@ -1,8 +1,14 @@
...
-#emphasis {
+.emphasis,
+a {
   color: violet;
 }
+
+a.active,
+a:hover {
+  color: rebeccapurple !important;
+}
```
Comment (Richie78321) [inline src/main/webapp/style.css:14]: nit: no newline

File: src/main/webapp/style.css [hunk 6]
```diff
@@ -1,8 +1,14 @@
...
 
-#emphasis {
+.emphasis,
+a {
   color: violet;
 }
+
+a.active,
+a:hover {
+  color: rebeccapurple !important;
```
Comment (billyeh) [inline src/main/webapp/style.css:13]: Just wondering why we need the !important?

File: src/main/webapp/style.css (hunk 4 above)
Comment (Obinnabii) [reply src/main/webapp/style.css:4]: Yeah def, I think this is here from rebasing on another branch. I’ll fix it!

File: src/main/webapp/dashboard.html (hunk 2 above)
Comment (Obinnabii) [reply src/main/webapp/dashboard.html:37]: I was planning on showing mentors/mentees that you have matched with her too

File: src/main/webapp/style.css (hunk 6 above)
Comment (Obinnabii) [reply src/main/webapp/style.css:13]: The color wasn't showing up without it for some reason. I think the bootstrap styling was overriding it.

File: src/main/webapp/dashboard.html (hunk 2 above)
Comment (Richie78321) [reply src/main/webapp/dashboard.html:37]: Oh okay. Then I think maybe the comment above that div is a bit misleading.

File: src/main/webapp/style.css (hunk 5 above)
Comment (Richie78321) [reply src/main/webapp/style.css:21]: Should have been more specific -- no newline at the end of the file.

Comment (Obinnabii) [reply src/main/webapp/style.css:21]:
```suggestion
}

```

**Pull request merged**